cmd/scaleComponent.go

```diff
@@ -0,0 +1,123 @@
+// Copyright © 2023
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"errors"
+	"strconv"
+
+	client2 "github.com/equinor/radix-cli/generated-client/client"
+	"github.com/equinor/radix-cli/generated-client/client/component"
+	"github.com/equinor/radix-cli/pkg/client"
+	"github.com/equinor/radix-cli/pkg/flagnames"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+// scaleCmd represents the scale command
+var scaleComponentCmd = &cobra.Command{
+	Use:   "component",
+	Short: "Scale component replicas",
+	Long: `Used for manually scaling up or down replicas of a Radix application component.
+Note: Manual scaling will persist across deployments, and will disable autoscaling.
+
+Examples:
```

Comment:
Please use the field `Example`

Reply:
fixed, also used the deprecation field 👍